specification/vi/resource-manager/readme.design.md

```diff
@@ -36,4 +36,4 @@ Users will be able to easily create, manage and delete a Azure Video Analyzer fo
 
 ## Type Model
 Resource provider namespace: Microsoft.VideoIndexer
-Resource type: accounts. <b><u>type model is still under develop.</u></b>. The resource type properties can be found [here](./Microsoft.VideoIndexer/preview/2022-04-13-preview/vi.json#L413)
\ No newline at end of file
+Resource type: accounts. <b><u>type model is still under develop.</u></b>. The resource type properties can be found [here](./Microsoft.VideoIndexer/stable/2022-08-01/vi.json#L413)
```

Comment:
I don't think we require this file in spec repo.  Is this file mainly about the description of your service?